chess/anomaly.py

```diff
@@ -0,0 +1,96 @@
+from collections import deque
```

Comment:
Are we sure we want to include this in CHESS at this point? I am still of the opinion that this belongs in anomaly detection for now.